indigo_app/static/javascript/indigo/grammars/slaw.js

```diff
@@ -0,0 +1,175 @@
+(function(exports) {
+  "use strict";
+
+  class SlawGrammarModel extends Indigo.grammars.GrammarModel {
+    constructor (...args) {
+      super(...args);
```

Comment:
should these `...`s come out?

Reply:
No, this is JS's equivalent of `foo(*args)` in python.

Reply:
got it, thanks!